components/feature/tabs/src/main/java/mozilla/components/feature/tabs/tabstray/TabsFeature.kt

```diff
@@ -24,17 +24,17 @@ class TabsFeature(
     tabsTray: TabsTray,
     private val store: BrowserStore,
     tabsUseCases: TabsUseCases,
-    thumbnailsUseCases: ThumbnailsUseCases,
     private val defaultTabsFilter: (TabSessionState) -> Boolean = { true },
-    closeTabsTray: () -> Unit
+    closeTabsTray: () -> Unit,
+    thumbnailsUseCases: ThumbnailsUseCases? = null
```

Comment:
Let's move this up a line or two.

In Kotlin, if the last parameter is a lambda expression, instead of doing this:

```kotlin
objectInstance.foo(someValue, { callbackValue ->
  doSomething(callbackValue)
})
```

You can write it as this:
```kotlin
objectInstance.foo(someValue) { callbackValue ->
  doSomething(callbackValue)
}
```

That is to say, you can take the last lambda expression and put it outside the function's parameters. The code is the exact same, it's just a nicer API to use.